WordPress/wordpress-playground
# Remove stale patches for PHP 8.1 and PHP 8.2

## What is this PR doing?

This PR removes the PHP 8.1 and 8.2 patches that are no longer required. They could actually have been removed a few months ago, but I never did that. Well, let's do it now.

It removes workarounds and brings us one step closer to source–code parity with PHP.

## Testing Instructions

* Confirm all PHP 8.1 and 8.2 builds were updated
* Confirm the unit tests and the E2E tests pass – they test different PHP features on all the shipped versions
